build.sbt

```diff
@@ -2,62 +2,78 @@ import sbt.Keys._
 
 import java.io.File
 
-def doSharedSetup(project: Project, folder: String) =
+def doServiceSetup(project: Project, folder: String) =
   project
     .in(new File(folder))
     .dependsOn(common % "compile->compile;test->test")
     .settings(Common.settings: _*)
     .settings(Finatra.settings: _*)
+    //I don't hink we use this anymore?
     .settings(Revolver.settings: _*)
     .settings(DockerCompose.settings: _*)
     .enablePlugins(DockerComposePlugin)
     .enablePlugins(JavaAppPackaging)
 
+def doSharedLibrarySetup(project: Project, folder: String) =
+  project
+    .in(new File(folder))
+    .settings(Common.settings: _*)
+    .settings(DockerCompose.settings: _*)
+    .enablePlugins(DockerComposePlugin)
+    .enablePlugins(JavaAppPackaging)
+
+def doSharedSierraSetup(project: Project, folder: String) =
+  doServiceSetup(project = project, folder = folder)
+    .dependsOn(sierra_adapter_common % "compile->compile;test->test")
+
 lazy val common = project
   .settings(Common.settings: _*)
   .settings(DockerCompose.settings: _*)
   .enablePlugins(DockerComposePlugin)
   .settings(libraryDependencies ++= Dependencies.commonDependencies)
 
-// We still inherit from the main common because we need the models it
-// contains (e.g. IdentifiedWork) to construct display models.
-lazy val common_display = doSharedSetup(project, "sbt_common/display")
+// It depends on common because it uses JsonUtil
+lazy val work_model = doSharedLibrarySetup(project, "sbt_common/work_model")
```

Comment:
I don't really like the naming scheme here, but it's not blocking to merge this PR.

Maybe something like:
- `uk.ac.wellcome.models.work.internal._`
- `uk.ac.wellcome.models.work.display._`